polls/models.py

```diff
@@ -12,13 +11,27 @@
 class Question(Page):
     subpage_types = ['polls.Choice']
 
+    def can_vote(user):
+        if PollVote.objects.filter(user=user) == []:
+            return True
+        else:
+            return False
+
     def choices(self):
         return Choice.objects.live().child_of(self)
 
 
 class Choice(Page):
     votes = models.IntegerField(default=0)
+    poll_votes = models.ManyToManyField('PollVote', related_name='set_vote',
+                                        null=True, blank=True)
 
     promote_panels = Page.promote_panels + [
         FieldPanel('votes'),
     ]
+
+
+class PollVote(models.Model):
```

Comment:
I think this should be `ChoiceVote` - no other place does the word polls come up.